src/components/FeatureFlagRenderer.js

```diff
@@ -1,30 +1,50 @@
 /* @flow */
 import { Component } from "react";
 
-import { ldOverrideFlag } from "../lib/utils";
-import { FeatureFlagType } from "../types/FeatureFlag";
+import { ldClientWrapper, ldOverrideFlag } from "../lib/utils";
 
-type Props = FeatureFlagType & { ldClientWrapper: Object };
+type Props = FeatureFlagType & ConfigType;
+type State = {
+  checkFeatureFlagComplete: boolean,
+  flagValue: any
+};
 
-export default class FeatureFlagRenderer extends Component {
-  props: Props;
-  state: {
-    checkFeatureFlagComplete: boolean,
-    flagValue: any
-  };
+export default class FeatureFlagRenderer extends Component<Props, State> {
+  _isMounted: boolean;
 
   constructor (props:Props) {
     super(props);
 
+    const { flagKey, clientOptions } = this.props;
+    const bootstrap = clientOptions && clientOptions.bootstrap;
+
     this.state = {
       checkFeatureFlagComplete: false,
-      flagValue: false
+      flagValue: bootstrap &&
+        typeof bootstrap === "object" &&
```

Comment:
This ended up passing type checking afterall?

Reply:
Yeah this solved the issue, flow just really wanted me to make sure to check the type.